pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -252,12 +251,6 @@ type podToMount struct {
 	mountRequestTime time.Time
 }
 
-const (
-	// Maximum errors to be stored per pod in desiredStateOfWorld.podErrors to
-	// prevent unbound growth.
-	maxPodErrors = 10
-)
-
```

Comment:
Why was this removed btw? I thought we wanted to avoid memory leak by storing errors infinitely. Does this PR solves that problem in a different way?